network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html

```diff
@@ -24,6 +24,10 @@
   {% endif %}
 {% endblock %}
 
+{% block description %}
+  <p class="tw-body tw-line-clamp-3"> {{page.specific.get_meta_description}} </p>
```

Comment:
The descriptions appear in English, but using `.localized` fixes this:

<img width="1145" alt="Capture d’écran 2022-06-03 à 12 23 42" src="https://user-images.githubusercontent.com/1294206/171837812-8e27e92b-3ae1-497f-adbd-885b660c0d32.png">


You should be able to reproduce with the prod DB on http://localhost:8000/fr/blog/mozilla-explains-ethical-ai-the-choice-is-not-yours-sometimes/

```suggestion
  <p class="tw-body tw-line-clamp-3"> {{page.specific.localized.get_meta_description}} </p>
```

Reply:
Great suggestion Theo! I have added this to the PR, thanks 😄